### ISSUE TITLE
False positive with UselessVariable

### ISSUE THREAD

**Issue by nunoperalta** [slevomat/coding-standard#918]
Did "composer update" to ensure I have the latest version, before posting this: 6.1.5

With the below, I don't get any errors:

```php
function getHeaderData1(string &$raw_headers) : string {
	$header = '';

	preg_match('/(^Header-1: (?:.*)(?:\n\s+(?:.*))*)/im', $raw_headers, $matches);
	if (!empty($matches) && !empty($matches[1])) $header .= $matches[1]."\n";
	preg_match('/(^Header-2: (?:.*)(?:\n\s+(?:.*))*)/im', $raw_headers, $matches);
	if (!empty($matches) && !empty($matches[1])) {
		$header .= $matches[1]."\n";
	}

	return $header;
}
```

With the below I get an error:

```php
function getHeaderData(string &$raw_headers) : string {
	$header = '';

	preg_match('/(^Header-1: (?:.*)(?:\n\s+(?:.*))*)/im', $raw_headers, $matches);
	if (!empty($matches) && !empty($matches[1])) $header .= $matches[1]."\n";
	preg_match('/(^Header-2: (?:.*)(?:\n\s+(?:.*))*)/im', $raw_headers, $matches);
	if (!empty($matches) && !empty($matches[1])) $header .= $matches[1]."\n";

	return $header;
}
```

SlevomatCodingStandard.Variables.UselessVariable.UselessVariable
Useless variable $header.

That's at the last "if (....) $header .= ..."  line.

**Comment by kukulich** [comment]
Yes, we don't support `if` without curly braces.
